docs/source/advanced_features/fusion_dataset.rst

```diff
@@ -1,10 +1,27 @@
-##########################
- Fusion Dataset Structure
-##########################
+################
+ Fusion Dataset
+################
+
+Fusion dataset represents datasets with data collected from multiple sensors.
+Typical examples of fusion dataset are some autonomous driving datasets,such as `nuScenes`_ and `KITTI-tracking`_.
+
+.. _nuScenes: https://gas.graviti.cn/dataset/motional/nuScenes
+.. _KITTI-tracking: https://gas.graviti.cn/dataset/data-decorators/KITTITracking
+
+
+Fusion dataset is one of the topmost concept in TensorBay format.
+Each fusion dataset includes a catalog and a certain number of fusion segments.
+
+The corresponding class of fusion dataset is :class:`~tensorbay.dataset.dataset.FusionDataset`.
+
+**************************
+ fusion dataset format
+**************************
 
 TensorBay also defines a uniform fusion dataset format.
```

Comment:
do not need `TensorBay also defines a uniform fusion dataset format. This topic explains the related concepts.`